packages/marko/src/runtime/components/ComponentDef.js

```diff
@@ -86,26 +88,37 @@ ComponentDef.prototype = {
 
 ComponentDef.prototype.nk = ComponentDef.prototype.___nextKey;
 
-ComponentDef.___deserialize = function(o, types, global, registry) {
+ComponentDef.___deserialize = function(o, typeName, registry) {
```

Comment:
Looks like this is Marko 4 code, this change should not be needed.

Reply:
This is a small cleanup, as we already have `typeName` we need when calling, and `global` is not used (dead code).